services/search/api.py

```diff
@@ -99,12 +100,33 @@ def to_representation(self, obj):
                 representation["department"] = DepartmentSerializer(obj.department).data
                 if obj.geometry:
                     representation["geometry"] = munigeo_api.geom_to_json(
-                            obj.geometry, DEFAULT_SRS
-                        )
+                        obj.geometry, DEFAULT_SRS
+                    )
                 else:
                     representation["geometry"] = None
-                    
-            if object_type == "address" or object_type == "unit":
+
+            if object_type == "address":
+                representation["number"] = getattr(obj, "number", "")
+                representation["number_end"] = getattr(obj, "number_end", "")
+                representation["letter"] = getattr(obj, "letter", "")
+                representation["modified_at"] = getattr(obj, "modified_at", "")
+                municipality = {}
+                municipality["id"] = getattr(obj.street, "municipality_id", "")
+                municipality["name"] = {}
+                municipality["name"]["fi"] = getattr(
+                    obj.street.municipality, "name_fi", ""
+                )
+                municipality["name"]["sv"] = getattr(
+                    obj.street.municipality, "name_sv", ""
+                )
+                representation["municipality"] = municipality
+                street = {}
+                street["name"] = {}
+                street["name"]["fi"] = getattr(obj.street, "name_fi", "")
+                street["name"]["sv"] = getattr(obj.street, "name_sv", "")
+                representation["street"] = street
+
+            if object_type == "unit" or object_type == "address":
```

Comment:
If-statements could be simplified here to a single statement

Reply:
Hmm, that would lead into dry code, the code below would have to be in both scopes if object_type == "unit":  and             if object_type == "address":   

```
 if obj.location:
                    representation["location"] = munigeo_api.geom_to_json(
                        obj.location, DEFAULT_SRS
                    )
```